### PR TITLE
feat: Implement get org route

### PR DESCRIPTION
**Pull request by levansuper**
# What kind of change does this PR introduce?
 

# What is the current behavior?
 

# What is the new behavior (if this is a feature change)?

# Other information

### CODE REVIEW

File: src/services/routes/org.routes.spec.ts [hunk 1]
```diff
@@ -330,4 +330,77 @@ describe('org routes', () => {
...
+      ['Non ASCII', NON_ASCII_ORG_NAME],
+    ])('%s name should return an org', async (_type, name: string) => {
+      const getOrgSuccessResponse = {
+        didSucceed: true,
+
+        result: {
+          name,
+          memberAccessType: 'INVITE_ONLY',
+        },
+      } as const;
```
Comment (gnarea) [inline src/services/routes/org.routes.spec.ts:350]: If we use `as const` here, we're creating a new type, which we don't want to do. Instead, we should explicitly set the type of this variable (`SuccessfulResult<Org...>`).

File: src/services/routes/org.routes.spec.ts [hunk 2]
```diff
@@ -330,4 +330,77 @@ describe('org routes', () => {
...
+
+      expect(mockGetOrg).toHaveBeenCalledWith(ORG_NAME, {
+        logger: serverInstance.log,
+        dbConnection: serverInstance.mongoose,
+      });
+      expect(response).toHaveProperty('statusCode', HTTP_STATUS_CODES.NOT_FOUND);
+      expect(response.json()).toHaveProperty('type', OrgProblemType.ORG_NOT_FOUND);
+    });
+
+    test('Malformed name should resolve into not found status', async () => {
```
[the remark this answers is not in the document]
Comment (gnarea) [reply src/services/routes/org.routes.spec.ts:387]: I don't think we need to test the malformed case here. What difference does it make?

**Review by gnarea — CHANGES_REQUESTED** [review]
Thanks! Nearly there

File: src/services/routes/org.routes.spec.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (levansuper) [reply src/services/routes/org.routes.spec.ts]: the only difference is in the input. The code behaves the same way so I think it's redundant

**Review by gnarea — APPROVED** [review]
Thanks! 🚀

**Pull request merged**